router.go

```diff
@@ -849,19 +849,24 @@ func (p *ControllerRegister) recoverPanic(context *beecontext.Context) {
 					return
 				}
 			}
-			var stack string
 			Critical("the request url is ", context.Input.URL())
 			Critical("Handler crashed with error", err)
-			for i := 1; ; i++ {
-				_, file, line, ok := runtime.Caller(i)
-				if !ok {
-					break
-				}
-				Critical(fmt.Sprintf("%s:%d", file, line))
-				stack = stack + fmt.Sprintln(fmt.Sprintf("%s:%d", file, line))
-			}
 			if BConfig.RunMode == DEV {
+				var stack string
+				for i := 1; ; i++ {
+					_, file, line, ok := runtime.Caller(i)
+					if !ok {
+						break
+					}
+					Critical(fmt.Sprintf("%s:%d", file, line))
+					stack = stack + fmt.Sprintln(fmt.Sprintf("%s:%d", file, line))
+				}
 				showErr(err, context, stack)
+				return
```

Comment:
in the dev mode, panic already output ctx.ResponseWriter.Write([]byte(body)). and when call `showErr(err, context, stack)` will that broken?